spark-submit.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+zip -r listenbrainz_spark.zip listenbrainz_spark/
+time ./run.sh /usr/local/spark/bin/spark-submit --master spark://spark-master.spark-network:7077 --num-executors=28 --executor-memory=1g --driver-memory=1g --py-files listenbrainz_spark.zip "$@"
```

Comment:
Before too long we will want to move these into a config file. For instance, once we move to azure, these values may no longer fit.

Reply:
The values will be moved to config once they are finalised.

Reply:
Can you please make a jira ticket for this, so it doesn't slip off our radar?